lib/carin_for_blue_button_test_kit/client/v2.0.0/endpoints/next_page_endpoint.rb

```diff
@@ -8,7 +8,7 @@ class NextPageEndpoint < Inferno::DSL::SuiteEndpoint
     include CarinForBlueButtonTestKit::MockAuthorization
 
     def test_run_identifier
-      extract_client_id_from_bearer_token(request) || extract_bearer_token(request)
+      extract_client_id_from_bearer_token(request)
```

Comment:
If you're removing the fallback to check the token value, you need to update the guidance in the input
<img width="555" alt="image" src="https://github.com/user-attachments/assets/4b179ba3-6c78-4043-aade-b681d408a203" />

Reply:
<img width="805" alt="image" src="https://github.com/user-attachments/assets/db516957-9f99-4fb4-ab2e-df1dd58ed88a" />

We can't tell the user to construct their own access token. The access token content and encoding scheme is private to the server (and as discussed, we intend to change it in the future anyway). Instead what you can do is provide the JWT in the subsequent wait dialog, like how we do [here](https://github.com/inferno-framework/davinci-dtr-test-kit/blob/main/lib/davinci_dtr_test_kit/client_groups/dinner_static/dtr_smart_app_dinner_questionnaire_package_request_test.rb#L119) in DTR.